client/landing/gutenboarding/gutenberg-types-patch.d.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import '@wordpress/compose';
+import '@wordpress/element';
+
```

Comment:
Was it necessary to import these?

Reply:
Declarations seems to be ignored by TS if there is no import/export inside the module. However only one would be enough so it doesn't matter what we import.

If I do an empty export like at the end of [this example(https://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-modifying-module-d-ts.html), typecheck is running fine but when building the app I get this error from webpack `Uncaught Error: Cannot find module '../../gutenberg-types-patch'`

Reply:
Nice find 👍